### PR TITLE
Form Implementation

### PR DESCRIPTION
**Pull request by BryanGK**
This PR implements the form as designed.

### CODE REVIEW

File: app/schemas/schema.js [hunk 1]
```diff
@@ -1,112 +1,230 @@
...
+    // "Last Mile Support",
+    // "Anticipated Projects",
+    // "Underserved Areas",
+    // "Satellite Proposal",
+    // "Capital Cost Funding",
+    // "Capital Cost Explanation",
+    // "Last Mile Broadband",
+    // "Last Mile Fixed Wireless",
+    // "Last Mile New Backbone",
+    // "Highway Cellular"
```
Comment (wenzowski) [inline app/schemas/schema.js:26]: Why are the required fields commented out?

File: app/schemas/schema.js [hunk 2]
```diff
@@ -1,112 +1,230 @@
...
+      "type": "object",
       "properties": {
-        "organization": {
-          "title": "Applicant organization (legal name):",
+        "Organization": {
+          "title": "Organization name (legal name)",
           "type": "string"
         },
-        "registrationNumber": {
+        "Registration Number": {
```
Comment (wenzowski) [inline app/schemas/schema.js:37]: Why change these from camelCase variable names to titles with spaces? These are the data keys that will be defined in our json object when the field is filled.

File: app/schemas/schema.js [hunk 3]
```diff
@@ -1,112 +1,230 @@
...
           "type": "string",
-          "title": "Band number, society number, or business registration number (if applicable):"
+          "title": "Band number, society number, or business registration number (optional)"
         },
-        "streetNumber": {
+        "Unit Number": {
           "type": "number",
-          "title": "Street number:"
+          "title": "Unit number (optional)"
```
Comment (wenzowski) [inline app/schemas/schema.js:43]: It makes sense to have the `(optional)` in the title for now, but let's get a ticket opened upstream? If a field isn't marked required then the toolkit should automatically add this label.

File: app/schemas/schema.js [hunk 4]
```diff
@@ -1,112 +1,230 @@
...
           "type": "string",
-          "title": "Postal code:"
+          "title": "Postal code (H0H 0H0)"
         }
       },
-      "type": "object"
     },
-    "Primary Contact": {
+    "Contact Information": {
+      "title": "Contact Information",
```
Comment (wenzowski) [inline app/schemas/schema.js:68]: Might we want a description of whose contact information we're collecting?

File: app/schemas/uiSchema.js [hunk 5]
```diff
@@ -1,22 +1,58 @@
 const uiSchema = {
-  "nonISPCollaborationPlan": {
-    "ui:widget": "textarea"
+  "Zone Feedback": {
+    "ui:widget": "textarea",
+    "ui:options": {
+      "cols": 100
```
Comment (wenzowski) [inline app/schemas/uiSchema.js:5]: is the cols section working? What about rows? If not let's get a ticket open

**Review by wenzowski — CHANGES_REQUESTED** [review]

File: app/schemas/schema.js (hunk 1 above)
Comment (BryanGK) [reply app/schemas/schema.js:26]: oh darn, that was for testing. I'll fix that.

File: app/schemas/schema.js (hunk 2 above)
Comment (BryanGK) [reply app/schemas/schema.js:37]: Ah ok, when I wasn't explicit with the "title" prop it looked wonky. But I'll go back to camelCase

File: app/schemas/schema.js (hunk 3 above)
Comment (BryanGK) [reply app/schemas/schema.js:43]: hmm have not seen any optional labels applied automatically.

File: app/schemas/schema.js (hunk 4 above)
Comment (BryanGK) [reply app/schemas/schema.js:68]: How so? I was implementing from the design.

File: app/schemas/uiSchema.js (hunk 5 above)
Comment (BryanGK) [reply app/schemas/uiSchema.js:5]: cols and rows not working, will open ticket

File: app/schemas/schema.js (hunk 3 above)
Comment (BryanGK) [reply app/schemas/schema.js:43]: https://github.com/button-inc/service-development-toolkit/issues/379

**Pull request merged**